### PR TITLE
Add TimeLine view to inspect tracked times

### PR DESCRIPTION
**Pull request by hotzenklotz**
### Mailable description of changes:
- [Admin.TimeTracking] Add a new time tracking view to inspect a user's tracked time spans for a given day. Time spans are plotted in a time line chart.

### Steps to test:
- Go to http://localhost:9000/timetracking
- Select a user and find a day with data

------
- [x] Ready for review

### CODE REVIEW

File: app/assets/javascripts/react_router.js [hunk 1]
```diff
@@ -297,6 +298,11 @@ class ReactRouter extends React.Component<Props> {
                   path="/help/keyboardshortcuts"
                   component={KeyboardShortcutView}
                 />
+                <SecuredRoute
+                  isAuthenticated={isAuthenticated}
+                  path="/timetracking"
```
Comment (normanrz) [inline app/assets/javascripts/react_router.js]: Make it a user-specific route? `/user/:user_id/timetracking`

File: app/assets/javascripts/admin/admin_rest_api.js [hunk 2]
```diff
@@ -442,3 +443,23 @@ export async function getDatastores(): Promise<Array<APIDatastoreType>> {
 export async function getActiveUser(options: Object = {}) {
   return Request.receiveJSON("/api/user", options);
 }
+
+// ### TimeTracking
+export async function getTimeTrackingForUserByDay(
+  userEmail: string,
+  day: Date,
+): Promise<Array<APITimeTrackingType>> {
+  const month = day.getMonth() + 1;
```
Comment (normanrz) [inline app/assets/javascripts/admin/admin_rest_api.js]: maybe use `moment(day).format("M")` etc.

Comment (normanrz) [reply app/assets/javascripts/admin/admin_rest_api.js]: for sanity

File: app/assets/javascripts/admin/views/time/time_line_view.js [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+                  optionFilterProp="children"
+                  style={{ width: "100%" }}
+                  onChange={this.handleUserChange}
+                >
+                  {this.state.users.filter(u => u.isActive).map((user: APIUserType) => (
+                    <Option key={user.id} value={user.email}>
+                      {`${user.lastName}, ${user.firstName} ${user.email}`}
+                    </Option>
+                  ))}
+                </Select>,
```
Comment (normanrz) [inline app/assets/javascripts/admin/views/time/time_line_view.js]: `,`

File: app/assets/javascripts/react_router.js (hunk 1 above)
Comment (hotzenklotz) [reply app/assets/javascripts/react_router.js]: Why should it be user specific if the user can be selected from the dropdown? I think admins would appreciate it if they can switch between users.

**Pull request merged**